files/error.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2019 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package files
```

Comment:
Are you sure that it is the right package for `HttpError`?

Reply:
I renamed package to utils. It basically contains different IO utils